microm.js

```diff
@@ -7,13 +8,13 @@ var config = require('./lib/config');
 
 config.setup();
 
-class Microm {
+class Microm extends EventEmitter {
   constructor() {
+    super();
     this.isRecording = false;
     this.recordRTC = null;
     this.player = null;
     this.mp3 = null;
-    this.eventListeners = {};
```

Comment:
This actually breaks the lib because the Player component still relies on it https://github.com/zzarcon/microm/blob/master/lib/player.js#L61. 

I really like the idea of using **EventEmitter** for the lib instead of a custom implementation of the event system, but this still needs to change other few things to make it work again.

Actually it's my fault for not having a proper testsuite that cover this :\ I have to fix it but will be great if you can fix the missing things :D